### PR TITLE
Bring the SurveyHero and GitHub versions more in sync

### PR DESCRIPTION
**Pull request by rylev**
cc @JasunC 

I also changed a few answers and questions on SurveyHero (mostly just punctuation) so that they match what's in the GitHub doc.

### CODE REVIEW

File: surveys/2021-annual-survey/questions.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Type: select one
 Type: select all that apply (optional)
 
 - I plan to return to using Rust in the future
-- I consider myself part of the Rust Community
+- I consider myself a part of the Rust community
```
Comment (nrc) [inline surveys/2021-annual-survey/questions.md:36]: It reads better without the 'a' to me, agree on the lower case c. Doesn't matter too much either way

File: surveys/2021-annual-survey/questions.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ Type: select all that apply (optional)
 Type: select all that apply (optional)
 
 - I plan to use Rust in the future
-- I consider myself part of the Rust Community
+- I consider myself a part of the Rust community
```
Comment (nrc) [inline surveys/2021-annual-survey/questions.md:57]: ditto

File: surveys/2021-annual-survey/questions.md [hunk 3]
```diff
@@ -668,7 +676,7 @@ Type: free form (optional)
 
 ## Rust in Education
 
-### Have you taken in the past year or are you currently taking a course or training which uses or teaches Rust?
+### Are you currently or have you in the last year taken a course or training which uses or teaches Rust?
```
Comment (nrc) [inline surveys/2021-annual-survey/questions.md:679]:
```suggestion
### Are you currently, or have you in the last year, taken a course or training which uses or teaches Rust?
```

**Review by nrc — APPROVED** [review]

File: surveys/2021-annual-survey/questions.md (hunk 1 above)
Comment (rylev) [reply surveys/2021-annual-survey/questions.md:36]: I think you're right. I'll change back

**Pull request merged**